core/src/main/java/org/elasticsearch/transport/RequestHandlerRegistry.java

```diff
@@ -77,6 +80,10 @@ public boolean isForceExecution() {
         return forceExecution;
     }
 
+    public boolean isCheckSizeLimit() {
```

Comment:
The name `isCheckSizeLimit` is not clear what it's for, but it's really used only to check whether or not the request being handled can cause the circuit breaker to trip so it should be implementation specific.